packages/@aws-cdk/aws-lambda/test/docker-lambda-handler/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM 628053151772.dkr.ecr.sa-east-1.amazonaws.com/awslambda/nodejs12.x-runtime-internal:beta
+FROM public.ecr.aws/lambda/nodejs:12 AS stage
```

Comment:
I believe I didn't have access to the previous repo, but this one worked. Added `AS stage` to use docker target in integration test.